lib/grizzly/zwave/commands/switch_multilevel_report.ex

```diff
@@ -44,9 +44,9 @@ defmodule Grizzly.ZWave.Commands.SwitchMultilevelReport do
   end
 
   def encode_value(:off), do: 0x00
-  def encode_value(value) when value in 0..100, do: value
+  def encode_value(value) when value in 0..99, do: value
   def encode_value(:unknown), do: 0xFE
-  def encode_value(:previous), do: 0xFF
+  def encode_value(0xFF), do: 99
```

Comment:
```suggestion
  def encode_value(0xFF), do: 0xFF
```

Reply:
0xFF, as a reported level value, means "100% hardware level", so the same as 99